src/stories/components/JsPsych/JsPsych.js

```diff
@@ -0,0 +1,240 @@
+/* eslint-disable */
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { observeStimuliCompletion } from '../../../firebase/api/gcp-utils';
+import { storeExperimentResult } from '../../../firebase/api/experiments';
+import example1 from '../../../images/example1.png';
+import example2 from '../../../images/example2.png';
```

Comment:
Have you tried it like this: `import { example2 } from '../../../images/example2.png';`